mirage/views/utils/index.ts

```diff
@@ -43,6 +43,10 @@ export function filter(model: ModelInstance, request: Request) {
                     if (field === 'id') {
                         return compareIds(model.id, val, toOperator(operator));
                     }
+                    if (val === '') {
```

Comment:
What if we are filtering a non-relationship field by empty string?

Reply:
Ahh, this is definitely too broad of a condition. I might put this after line 52 and update it so it's checking something like 
```
const actualValue = model[`${field}Ids`] || model[`${field}Id`];
if (actualValue === null) {
    return compareIds('', val, toOperator(operator));
```

Reply:
Actually, I feel like this would only be working by happenstance. Poking around a little bit in the console showed that 
```
x = undefined || null; // null
y = null || undefined; //undefined
```
In this specific case `${field}Ids` is `undefined` and `${field}Id` is `null` (like in `x = ...`, but it's possible that these are switched like in `y = ...`
so maybe an approach where we check if either one of them is `null`